service/routers/users.js

```diff
@@ -0,0 +1,50 @@
+import { Router } from 'express'
+import { User } from '../models/User.js'
+
+const usersRouter = Router()
+
+const getUsers = async (req, res) => {
+  if(req.user.role == "Admin" || req.user.role == "SuperUser") {
+    const users = await User.find({})
+    res.send(users)
```

Comment:
This will expose data about the user to the requestor that isn't safe, including the password hash.